llm-chain/src/templates.rs

```diff
@@ -35,8 +35,10 @@ pub struct PromptTemplate {
 
 impl PromptTemplate {
     /// Create a new prompt template from a string.
-    pub fn new(template: String) -> PromptTemplate {
-        PromptTemplate { template }
+    pub fn new(template: &str) -> PromptTemplate {
```

Comment:
```suggestion
    pub fn new<K: Into<String>>(template: K) -> PromptTemplate {
      PromptTemplate {
        template: template.into()
      }
   }
```
What do you think of that?